rust_src/src/windows.rs

```diff
@@ -1252,6 +1252,24 @@ pub fn window_next_buffers(window: LispWindowLiveOrSelected) -> LispObject {
     win.next_buffers
 }
 
```

Comment:
Close. Both lisp_fns need `min = "0"` because the DEFUN says "0, 1, 0" which means min 0, max 1, not interactive.

Reply:
oh, yeah, thx. Read porting guide poorly

Reply:
In rust_src/admin there is a script called rustize-emacs. C-u M-| is a good way to call it.

Reply:
added `min`'s